js/app.js

```diff
@@ -1,9 +1,62 @@
-// wait for DOM to load before running JS
+//wait for DOM to load before running JS
 $(document).ready( function() {
-
   // check to make sure JS is loaded
   console.log('JS is loaded!');
+  let $results = $("#results");  //cache the DOM search
```

Comment:
Hey Chen,

Great job on this homework! Also glad to hear you were mostly comfortable with the material. 

I would like to start seeing you commit more often, however. Committing early and often is not just best practice but really helps with bugs when we start writing more and more lines of code across different files! 

Matt